tests/src/e2e/setup.rs

```diff
@@ -667,7 +667,7 @@ where
         .env("ANOMA_LOG", "info")
         .env("ANOMA_TM_STDOUT", "true")
         .env("TM_LOG_LEVEL", "info")
-        .env("ANOMA_LEDGER__ETHEREUM__MODE", "Off")
+        .env("ANOMA_LEDGER__ETHEREUM_BRIDGE__MODE", "Off")
```

Comment:
eventually this should be removed, it's a bit hacky to include it here

Reply:
agreed, have opened https://github.com/anoma/namada/issues/665

We will need to revisit this soon probably as for some e2e tests, we will want to enable some components of the Ethereum bridge e..g `ANOMA_LEDGER__ETHEREUM_BRIDGE__MODE='EventsEndpoint'`

Reply:
a pr I have open allows this to be configured on demand https://github.com/anoma/namada/pull/646